taskfile/node_byte.go

```diff
@@ -0,0 +1,49 @@
+package taskfile
+
+import (
+	"github.com/go-task/task/v3/internal/execext"
+	"github.com/go-task/task/v3/internal/filepathext"
+)
+
+// A ByteNode is a node that reads a taskfile direct from a []byte object.
+type ByteNode struct {
+	*baseNode
+	data []byte
+}
+
+func NewByteNode(data []byte, dir string) (*ByteNode, error) {
+	return &ByteNode{
+		baseNode: NewBaseNode(dir),
+		data:     data,
+	}, nil
+}
+
+func (node *ByteNode) Location() string {
+	return "__bytes__"
+}
+
+func (node *ByteNode) Remote() bool {
+	return true
```

Comment:
Here, I'm not certain if this should actually be false. It may not matter either way.